frontend/ui/uimanager.lua

```diff
@@ -531,14 +532,14 @@ function UIManager:close(widget, refreshtype, refreshregion, refreshdither)
 end
 
 -- schedule an execution task, task queue is in ascendant order
-function UIManager:schedule(time, action, ...)
+function UIManager:schedule(time_fts, action, ...)
     local p, s, e = 1, 1, #self._task_queue
     if e ~= 0 then
         -- do a binary insert
         repeat
-            p = math.floor(s + (e - s) / 2)
-            local p_time = self._task_queue[p].time
-            if time > p_time then
+            p = math.floor((e + s) / 2) -- Not necessary to use (s + (e -s) / 2) here!
```

Comment:
Fun fact: manual factorization may make stuff slower in LuaJIT (because that shit is weird ;o).

Reply:
I don't see a factorisation here? 

( The `s + (e-s)/2` is a trick to avoid integer overflow on really large arrays. But 1st our array is clearly small enough and 2nd we have Lua numbers here. -> so no overflow)

Reply:
Possibly wrong terminology on my part (I don't speak maths :p).